src/github.com/matrix-org/dendrite/common/types.go

```diff
@@ -46,6 +46,16 @@ type DisplayName struct {
 // recognized by strconv.ParseBool
 type WeakBoolean bool
 
+// MTag contains the data for a Tag which can be referenced by the Tag name
+type MTag struct {
```

Comment:
Worth mentioning tags don't have to start with `m.`.

Reply:
well the spec sheet said m.tag so I thought....
Should I change it now? Or its fine?

Reply:
Ah my bad, I was interpreting this as as tag's names don't have to start with `m.`, but that's obviously not a problem here, as you're referencing the event type `m.tag`.